### ISSUE TITLE
'brew install gitfs' and 'brew info gitfs' - "Error: wrong number of arguments (0 for 1..3)"

### ISSUE THREAD

**Issue by eschen42** [Linuxbrew/homebrew-core#2789]
# Please always follow these steps:
- [x] Confirmed this is a problem with `brew install`ing one, specific formula and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Linuxbrew/brew/issues/new.
- [x] Attempt to replicate the problem on macOS with Homebrew if possible. **Package installs successfully on macOS/Homebrew**, but *it takes forever to do ANYTHING with my old mac!*.
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed all issues and retried your prior step?
- [x] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [x] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

**Please note we may immediately close your issue without comment if you delete or do not fill out the issue checklist and provide ALL the requested information.**

To help us debug your issue please explain:
- What you were trying to do (and why)
  - I was trying to install the gitfs formula (in a Docker container) to try out gitfs
  - `brew install gitfs` failed
    - this succeeded on mac OS
    - this also failed on my Ubuntu 16.04.2 LTS (xenial) box
      - rather than determine whether something is wrong with my box, I decided to try it in Docker
      - **the Docker container behavior was identical to what I experienced on my Linux box**  
  - `brew info gitfs` failed 
    - this happened in the following context, which may point to the problem faster than reviewing my gist
```
linuxbrew@78f515fa1f4d:~$ brew info gitfs
gitfs: stable 0.4.5.1, HEAD
Version controlled file system
https://www.presslabs.com/gitfs
Not installed
From: https://github.com/Linuxbrew/homebrew-core/blob/master/Formula/gitfs.rb
==> Dependencies
Required: libgit2 ✔
==> Requirements
Error: wrong number of arguments (0 for 1..3)
```
- What happened (include command output)
  - Please see above and my gist
- What you expected to happen
  - I expected `brew install` and `brew info` to succeed.
- Step-by-step reproduction instructions (by running `brew install` commands)
Please see my gist (https://gist.github.com/eschen42/eb5bade736c7883ccbe9b80e06f58201)
# New Formulae
To get a new formula into Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md).
To get a new formula that is a dependency of a formula in Linuxbrew/homebrew-core submit a Pull Request to https://github.com/Linuxbrew/homebrew-core/compare
To get a new formula that is not a dependency of a formula in Linuxbrew/homebrew-core submit a Pull Request to https://github.com/Linuxbrew/homebrew-extra/compare

**Comment by sjackman** [comment]
@rwhogg Is this issue resolved in `brew` master? I'm planning to tag `brew` 1.2.0 today.
@eschen42 Is using stable `brew` 1.1.13.

**Comment by rwhogg** [comment]
@sjackman, yes, this issue is resolved in `brew` master.

Note however that even in `brew` master, the formula doesn't install under superenv for unrelated reasons. It does install under stdenv.

**Comment by sjackman** [comment]
Thanks, Bob. `brew install gitfs` worked for me just now on `CentOS Linux release 7.1.1503`.
@eschen42 This bug will be resolved once `brew` 1.2.0 is released.

**Comment by sjackman** [comment]
Brew 1.2.0 is released! `brew update && brew install gitfs` should resolve your issue.
